corebuild/dotnet.ps1

```diff
@@ -10,12 +10,11 @@ if ((Test-Path $bootStrapperPath) -eq 0)
         mkdir $toolsLocalPath | Out-Null
     }
 
-    cp "bootstrap.ps1" $bootStrapperPath
+    cp (Join-Path $PSScriptRoot "bootstrap.ps1") $bootStrapperPath
 }
 
-
 # now execute it
-& $bootStrapperPath (Get-Location) $toolsLocalPath | Out-File (Join-Path (Get-Location) "bootstrap.log")
```

Comment:
Change `Join-Path (Get-Location) "bootstrap.log") ` to `Join-Path $PSScriptRoot "bootstrap.log"` for consistency